include/zephyr/sys/uuid.h

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (c) 2024, SECO Mind Srl
```

Comment:
There are some mixed copyright years/owners, are all of them accurate?

Reply:
I'll check them